core/storage/suggestion/gae_models.py

```diff
@@ -54,6 +54,9 @@
 SUGGESTION_TYPE_TRANSLATE_CONTENT = 'translate_content'
 SUGGESTION_TYPE_ADD_QUESTION = 'add_question'
 
+# Constants to define voiceover application target type.
+VOICEOVER_APPLICATION_TO_EXPLORATION = 'voiceover_exploration'
```

Comment:
Let's probably make this standardized for future application types which we might add. So I would suggest something along the lines of 
```APPLICATION_TYPE_VOICEOVER_FOR_EXPLORATION```

Any future constants would be of the form 
```APPLICATION_TYPE_{item}_FOR_{target}```

Reply:
> APPLICATION_TYPE_VOICEOVER_FOR_EXPLORATION

We are going to use these constants only for a voiceover application.

> APPLICATION_TYPE_{item}_FOR_{target}

The `item` is not going to change in the future!

Reply:
Hmmm, ok then, I'm fine with the current naming, but are you sure we won't be needing other forms of applications?

Reply:
> are you sure we won't be needing other forms of applications?

Right now I don't see anything similar to voiceover where the entire big task is dependent on one person, so I don't think we will be in need of an application procces in the future in other forms.

Reply:
It's hard to see the future. I would suggest APPLICATION_TYPE_VOICEOVER_FOR_EXPLORATION as the contstant name: it's clearer and more generalizable, and it's easier to see what the "noun" is  (APPLICATION_TYPE in this case).